committee-steering/governance/sig-governance.md

```diff
@@ -69,6 +69,12 @@ Subproject Owner Role.  (this different from a SIG or Organization Member).
   - Number: 2-3
   - Membership tracked in [sigs.yaml]
 
+- Subproject Leads
+  - Track and coordinate work
```

Comment:
I'd prefer to first discuss subproject management best practices, and then figure out what roles are needed.

We can use this as a starting point:
https://github.com/kubernetes/community/blob/master/sig-cluster-lifecycle/grooming.md